package.json

```diff
@@ -30,6 +30,7 @@
   },
   "scripts": {
     "start": "webpack-dashboard -- webpack serve --config webpack.config.js --env env=dev",
+    "docker:start": "webpack serve --config webpack.config.js --env env=dev",
```

Comment:
@devpurohit update command to support `webpack-dashboard` plugin:
```
webpack-dashboard -- webpack serve --config webpack.config.js --env env=dev
```

Reply:
Doesn't make a difference since the dashboard won't run in `docker-compose up`. Hence, resolving this change.